src/thunderbots/software/util/canvas_messenger/canvas_messenger.h

```diff
@@ -96,24 +210,46 @@ namespace Util
             uint8_t result[2];
         };
 
-       private:
+        // The number of pixels per meter
+        static const int PIXELS_PER_METER = 100;
+
+        // Colors
+        static constexpr Color FIELD_COLOR         = {0, 153, 0, 255};
+        static constexpr Color DEFENSE_AREA_COLOR  = {242, 242, 242, 255};
+        static constexpr Color FIELD_LINE_COLOR    = {242, 242, 242, 255};
+        static constexpr Color BALL_COLOR          = {255, 153, 0, 255};
+        static constexpr Color FRIENDLY_TEAM_COLOR = {230, 230, 0, 255};
+        static constexpr Color ENEMY_TEAM_COLOR    = {0, 230, 230, 255};
```

Comment:
Hmmm. So I think this is a bit of a tricky one. I totally see where you're coming from on this, but I rather like having all the draw functions as part of the `CanvasMessenger` class, and I don't see us adding a ton more then this. 

@MathewMacDougall - could I please get a second opinion on this?

Reply:
Assuming this is also referring to the "thunderbots specific" drawing functions above too.

I think I agree with Gareth and prefer leaving all the drawing functions together here, since there aren't too many, they do share similarities so still make some sense to have together, and is slightly simpler to use for whoever is drawing stuff (they don't have to remember 2 namespaces / paths).

That being said, I think it's reasonable to put these color constants somewhere else, like our existing `constants` file, or a new `colors` file, since they don't need to be coupled directly to the `CanvasMessenger`